tests/nat/middleware/test_middleware_components.py

```
@@ -45,6 +44,16 @@ def __init__(self, *, test_param: str, call_order: list[str]):
         self.test_param = test_param
         self.call_order = call_order
 
+    @property
+    def enabled(self) -> bool:
+        return True
+
+    async def pre_invoke(self, context):
```

Ruff (0.14.8)

Unused method argument: `context`

(ARG002)